pkg/patterns/declarative/pkg/manifest/objects.go

```diff
@@ -331,17 +332,18 @@ func ParseObjects(ctx context.Context, manifest string) (*Objects, error) {
 		out := &unstructured.Unstructured{}
 		err := decoder.Decode(out)
 		if err != nil {
-			log.WithValues("yaml", yaml).Error(err, "error decoding object")
-			return nil, fmt.Errorf("error decoding object: %v", err)
+			log.V(5).Info("Unable to parse into Unstructured, Storing as blob")
```

Comment:
@SomtochiAma 
Do you have some policies of choice "5" in `V`?

@johnsonj 
Do you have some policies of choose of number in `V`?
I checked what numbers are used in current code.
There are `V(1)` and `V(2)` only.

```shell
~/go/src/sigs.k8s.io/kubebuilder-declarative-pattern$ grep "V(" -r ./pkg
./pkg/patterns/declarative/image.go:				log.WithValues("manifest", manifestItem).WithValues("registry", registry).V(1).Info("applying image registory to manifest")
./pkg/patterns/declarative/image.go:				log.WithValues("manifest", manifestItem).WithValues("secret", secret).V(1).Info("applying image pull secret to manifest")
./pkg/patterns/declarative/pkg/manifest/objects.go:			log.V(5).Info("Unable to parse into Unstructured, Storing as blob")
./pkg/patterns/declarative/pkg/manifest/patch.go:		log.WithValues("patched", patched).V(2).Info("applying patches")
./pkg/patterns/declarative/pkg/kubectlcmd/client.go:	log.WithValues("stdout", stdout.String()).WithValues("stderr", stderr.String()).V(2).Info("ran kubectl apply")
./pkg/patterns/declarative/sort.go:			log.WithValues("group", o.Group).WithValues("kind", o.Kind).V(2).Info("unknown group / kind")
./pkg/patterns/declarative/labels.go:			log.WithValues("object", o).WithValues("labels", labels).V(1).Info("add labels to object")
./pkg/patterns/addon/patch.go:		log.WithValues("patch", patch).V(1).Info("parsed patch")
./pkg/patterns/addon/pkg/status/aggregate.go:			log.WithValues("type", gk).V(2).Info("type not implemented for status aggregation, skipping")
./pkg/patterns/addon/pkg/status/aggregate.go:	log.WithValues("object", src).WithValues("status", status).V(2).Info("built status")
```

Reply:
I will change it to 2 then. I was thinking it had the same numbers as `klog`